app/src/test/java/com/example/rocketreserver/LaunchListBlogKtTest.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.rocketreserver
+
+import org.junit.jupiter.api.Assertions.*
+
+internal class LaunchListBlogKtTest
```

Comment:
This isn't used

Reply:
Where these test files added automatically by code generation?